### PR TITLE
added screenshot to Seeker README

### PR DESCRIPTION
**Pull request by j-koreth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Seeker 1.0 Beta 6
```
Comment (humdingerb) [inline README.md]: Please remove the version from the title and use markdown formatting to make it a proper heading.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
+![Seeker screenshot](Seeker.png "Seeker")
```
Comment (humdingerb) [inline README.md]: Move the screenshot below the introductory sentence. Looks just better...

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by j-koreth** [comment]
Done! Any other changes?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
-Seeker 1.0 Beta 6
+Seeker 1.0
```
Comment (humdingerb) [inline README.md]: Remove the whole version, 1.0 too. :)

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Comment by j-koreth** [comment]
I'm an idiot

**Review by humdingerb — APPROVED** [review]

**Comment by j-koreth** [comment]
Sorry for the excessive messages, my browser glitched for some reason.

**Comment by humdingerb** [comment]
Thanks. You're not an idiot, even when repeating it a dozen times... :)

**Pull request merged**